src/main/java/streamingapi/client/StreamMonitor.java

```diff
@@ -0,0 +1,28 @@
+package streamingapi.client;
+
+/**
+ * Monitor used stop the execution or to check if the execution should continue.
+ *
+ * @author dancojocaru
+ */
+public class StreamMonitor {
+
```

Comment:
I see that this value is not customised from the tests, it isn't  necessary ?

Reply:
The default value is true, so it wont require initialisation. I'm not sure what do you mean by "customised".

Reply:
Got it. Customised in the sense that is not used from the test with a different value